packetbeat/flows/flows.go

```diff
@@ -0,0 +1,253 @@
+package flows
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/packetbeat/config"
+	"github.com/elastic/beats/packetbeat/publish"
+)
+
+type Flows struct {
```

Comment:
Looking at this file, I was thinking if it could be "cleaner" to split up the struct and its methods across multiple files.